sci-mathematics/calc/calc-2.12.5.3.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="C-style arbitrary precision calculator"
+DESCRIPTION="Calc is an interactive calculator which provides for easy large \
+numeric calculations, but which also can be easily programmed for difficult or \
+long calculations. It can accept a command line argument, in which case it \
+executes that single command and exits. Otherwise, it enters interactive mode. \
+In this mode, it accepts commands one at a time, processes them, and displays \
+the answers. In the simplest case, commands are simply expressions which are \
+evaluated."
+HOMEPAGE="http://www.isthe.com/chongo/tech/comp/calc/"
+COPYRIGHT="1999-2016 Landon Curt Noll"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.isthe.com/chongo/src/calc/calc-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0fcf60917efb10aab145f1180c278a3ab63d789aad6d5e5f066a4458b5f9b692"
+PATCHES="calc-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	calc$secondaryArchSuffix = $portVersion
+	cmd:calc$secondaryArchSuffix = $portVersion
+	cmd:calc_static$secondaryArchSuffix = $portVersion
+	cmd:cscript$secondaryArchSuffix = $portVersion
+	lib:libcalc$secondaryArchSuffix
+	lib:libcustcalc$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	calc${secondaryArchSuffix}_devel = $portVersion
+	devel:libcalc$secondaryArchSuffix
+	devel:libcustcalc$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	calc$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:awk
+	cmd:sed
+	cmd:diff
+	cmd:egrep
+	cmd:ctags
+	cmd:xargs
+	cmd:cmp
+	cmd:touch
+	cmd:less
+	cmd:strip
+	"
+
+BUILD()
+{
+	make all \
+		BLD_TYPE=calc-static-only \
+		LCC=gcc \
+		LN="ln -s" \
+		LDCONFIG= \
+		INCDIR=$includeDir \
+		LIBDIR=$libDir \
+		BINDIR=$binDir \
+		CALC_SHAREDIR=$dataDir/calc \
+		MANDIR=$manDir/man1 \
+		EXTRA_CFLAGS="-DUSE_TERMIOS -DHAVE_UID_T"
+}
+
+INSTALL()
+{
+	make install \
+		BLD_TYPE=calc-static-only \
+		LCC=gcc \
+		LN="ln -s" \
+		LDCONFIG= \
+		INCDIR=$includeDir \
+		LIBDIR=$libDir \
+		BINDIR=$binDir \
+		CALC_SHAREDIR=$dataDir/calc \
+		MANDIR=$manDir/man1 \
+		EXTRA_CFLAGS="-DUSE_TERMIOS -DHAVE_UID_T"
+
+	prepareInstalledDevelLibs \
+		libcalc libcustcalc
```

Comment:
After all, maybe it is better to have a single line:
```
	prepareInstalledDevelLibs libcalc libcustcal
```
Leaving the command alone and then putting two libs together in the same line is not very common.
(The reason why we sometimes put one library per line is when we have too many or when their names match a nice pattern.)